### PR TITLE
Documents POST account/settings/managed-enable

### PR DESCRIPTION
**Pull request by Dorthu**

### CODE REVIEW

File: src/data/endpoints/account.yaml [hunk 1]
```diff
@@ -53,6 +53,23 @@ endpoints:
...
+    authenticated: true
+    description: >
+      Enable Linode Managed on youy account.
+    methods:
+      POST:
+        money: true
+        scopes: account:modify
+        description: >
+          Enable Linode Managed on youy account. Only unrestricted users can do this.
+          See https://linode.com/managed for details and pricing information.
```
Comment (Dorthu) [inline src/data/endpoints/account.yaml]: I would really like to make this a link, but I couldn't figure out how to/find an example of doing so.  Anyone know how to do this? `<ExternalLink to="">` didn't work, `<a href="">` didn't work, and a `seeAlso`didn't do what I wanted.

File: src/data/endpoints/account.yaml [hunk 2]
```diff
@@ -53,6 +53,23 @@ endpoints:
             my_settings.country = 'US'
             my_settings.email = 'jsmith@company.com'
             my_settings.save()
+  /account/settings/managed-enable:
+    group: Settings
+    authenticated: true
+    description: >
+      Enable Linode Managed on youy account.
```
Comment (yoshizzle) [inline src/data/endpoints/account.yaml]: your account.

File: src/data/endpoints/account.yaml [hunk 3]
```diff
@@ -53,6 +53,23 @@ endpoints:
...
+    methods:
+      POST:
+        money: true
+        scopes: account:modify
+        description: >
+          Enable Linode Managed on youy account. Only unrestricted users can do this.
```
Comment (yoshizzle) [inline src/data/endpoints/account.yaml]: your account.

File: src/data/endpoints/account.yaml (hunk 2 above)
Comment (yoshizzle) [reply src/data/endpoints/account.yaml]: type: Action?

File: src/data/endpoints/account.yaml [hunk 4]
```diff
@@ -53,6 +53,25 @@ endpoints:
...
+    authenticated: true
+    description: >
+      Enable Linode Managed, our 24/7 monitoring service, on you account.
+    methods:
+      POST:
+        money: true
+        scopes: account:modify
+        type: Action
+        description: >
+          Enable Linode Managed, our 24/7 monitoring service, on you account.
```
Comment (jamesottinger) [inline src/data/endpoints/account.yaml]: ^ wording change needed

File: src/data/endpoints/account.yaml [hunk 5]
```diff
@@ -53,6 +53,25 @@ endpoints:
             my_settings.country = 'US'
             my_settings.email = 'jsmith@company.com'
             my_settings.save()
+  /account/settings/managed-enable:
+    group: Settings
+    authenticated: true
+    description: >
+      Enable Linode Managed, our 24/7 monitoring service, on you account.
```
Comment (jamesottinger) [inline src/data/endpoints/account.yaml]: ^ and there

**Pull request merged**